### PR TITLE
Fixes #6946: Document scheduler generic method

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/6946

### CODE REVIEW

File: tree/30_generic_methods/schedule_simple_catchup.cf [hunk 1]
```diff
@@ -17,11 +17,14 @@
 #####################################################################################
 
 # @name Schedule Simple Catchup
-# @description Trigger a repaired outcome when a job should be run
+# @description Trigger a repaired outcome when a job should be run (avoid loosing a job)
```
Comment (amousset) [inline tree/30_generic_methods/schedule_simple_catchup.cf]: losing

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**